_integration-schemas/linkedin-ads/v1/linkedin-ads-foreign-keys.md

```diff
@@ -0,0 +1,99 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas-foreign-keys/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+tap-reference: "integration"
```

Comment:
Don't forget to add `integration.name` here - otherwise you won't see any of your foreign keys:

```suggestion
tap-reference: "linkedin-ads"
```